package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Read a protobuf schema from the disk, parse it and resolve all imports",
   "main": "index.js",
   "dependencies": {
+    "lodash": "^4.8.2",
```

Comment:
lodash is more or less a standard library. Furthermore, implementing and properly testing the logic of  `intersectionBy` `groupBy` `differenceBy` `extend` `reduce` `unionBy` and `filter` is out of scope of this story.